modules/ROOT/pages/database-administration/composite-databases/list-composite-databases.adoc

```diff
@@ -0,0 +1,54 @@
+:description: This page describes how to list composite databases.
+[role=enterprise-edition not-on-aura]
+[[composite-databases-list]]
+= Listing composite databases
```

Comment:
I wonder if we should make all headings consistent - using the `-ing` or `infinitive` form.
```suggestion
= List composite databases
```